### PR TITLE
Add list API call to jujulib's bag of tricks.

### PR DESCRIPTION
**Pull request by kadams54**

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ module.exports = function(config) {
     // with the preprocessors, to watch the source files and serve the compiled
     // files.
     files: [
-      'jujugui/static/gui/build/app/assets/javascripts/jujulib/*.js',
```
Comment (kadams54) [inline karma.conf.js]: Without this change, any code updates in jujulib would need a `make clean` in order to be picked up by a test run.

**Comment by kadams54** [comment]
## QA

This PR just exposes an API endpoint; there's no GUI code that uses it (yet), so QA is the unit test. Make sure the code looks good.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4368/
Test PASSed.

**Comment by hatched** [comment]
:+1:

**Comment by kadams54** [comment]
:shipit: Thanks!

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**